src/routers/cycles.ts

```diff
@@ -1,7 +1,6 @@
 import type { PostgresJsDatabase } from 'drizzle-orm/postgres-js';
 import { default as express } from 'express';
 import type * as db from '../db';
-import { getGroups } from '../services/groups';
```

Comment:
shouldnt we keep the following router as we added groups back in? 

import { getGroups } from '../services/groups';

Reply:
added back, groups are now part of users router to get the groups related to a user and the groups router to get all groups